exercises/handling-errors/practice/src/shared.ts

```diff
@@ -0,0 +1,49 @@
+export const TASK_QUEUE_NAME = 'pizza-tasks';
```

Comment:
I know we don't use code doc, but this is where production work would definitely use code doc. Just needed to vent. Ignore this comment.